src/installer.js

```diff
@@ -1,15 +1,16 @@
 /* eslint-disable no-console, consistent-return,no-useless-escape */
 const fs = require('fs');
 const path = require('path');
-
 const util = require('util');
 
 const resolve = require('resolve');
-
 const spawn = require('cross-spawn');
-
 const JSON5 = require('json5');
 
+const { green, yellow } = require('colorette');
+
+const prompt = require('./utils/prompt');
```

Comment:
We can use `method` :smile:

Reply:
can you clarify?

Reply:
I think he means, using it as `this.prompt` instead of creating separate util and importing.

Reply:
Yep :+1:

Reply:
we don't have access to plugin class in the installer so I've inlined the util in installer instead